Readme.md

```diff
@@ -36,6 +36,9 @@ PunktDe:
         # or an eel expression like seen below
         sorting: title
 
+        # Optional: Trigger sorting only, when condition is met.
```

Comment:
# Optional: Trigger sorting only, when condition is met. Can be used to make sure that required properties are set as expected.

Reply:
done